examples/aks-spring-music/cnab/app/run

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+exec /cnab/app/porter-runtime run -f /cnab/app/porter.yaml
```

Comment:
I remember why this is checked in. We wan to generate this during build, but currently it's only generated during porter create.

Reply:
Yep, will do a round of cleanup on these once we make those big old changes.